etna/articles/blocks.py

```diff
@@ -197,6 +197,7 @@ class FeaturedCollectionBlock(SectionDepthAwareStructBlock):
     description = blocks.TextBlock(max_length=200)
     items = PageListBlock(
         "articles.ArticlePage",
+        "articles.RecordArticlePage",
```

Comment:
Do these two values need to be in a list?

Reply:
I don't think so, it works as is and when I tried putting it into a list the list wasn't getting resolved and threw an error

Reply:
https://github.com/nationalarchives/ds-wagtail/blob/develop/etna/core/blocks/page_list.py#L21-L29

Reply:
So it does :) I'm having a brain-fart.